cfgen/cfgen

```diff
@@ -75,7 +75,7 @@ nodes:
     data_iface: <str>  # name of network interface; e.g., eth1, eth1:c1
     data_iface_type: tcp|o2ib  # type of network interface;
                                # optional, defaults to "tcp"
-    m0_servers:
+    m0_servers:        # optional for client-only nodes
```

Comment:
This comment is ambiguous.  It can be interpreted as if `m0_servers` is only applicable to client-only nodes.
```suggestion
    m0_servers:        # optional
```